test/test_entrypoint.py

```diff
@@ -15,7 +15,7 @@
 
 """Unit tests for the `create_cc_sysroot.py` script."""
 
-from cross_compile.ros2_cross_compile import parse_args
+from cross_compile.ros_cross_compile import parse_args
```

Comment:
Aren't we removing all ros-deps?

Reply:
This is importing a module from this package - i renamed the source file which is why this line is changing